src/pages/signup-affiliates-details/index.js

```diff
@@ -34,7 +34,7 @@ const Overlay = styled.div`
 
 const affiliateSignupDetails = () => {
     const [is_popup_shown, setPopupShown] = useState(false)
-    const [is_error_message, setErrorMessage] = useState(false)
+    const [error_message, setErrorMessage] = useState('')
```

Comment:
[Need Clarifications]: Is it going to have a String or a boolean value?
if it is a string then you need to localize it.
if it is a boolean then better to change the default value to `false` and change the name to `has_error_message`.

Reply:
This is a String value. Used `localize` wrapper for this.